docs/source/release/v3.10.0/muon.rst

```diff
@@ -17,6 +17,10 @@ Algorithms
 Fit Functions
 -------------
 
+Bugfixes
+--------
+- Fixed a bug that menat transverse field asymmetry data was normalized to bin width. 
```

Comment:
Just to be clearer - I meant under the `Interfaces` heading within `muon.rst`. It does not really matter so much though, it should get tidied up when someone goes through the release notes before the next release.